lib/absinthe/phase/document/validation/selected_current_operation.ex

```diff
@@ -28,15 +30,36 @@ defmodule Absinthe.Phase.Document.Validation.SelectedCurrentOperation do
   end
 
   # Generate the error
-  @spec error :: Phase.Error.t()
-  defp error do
+  @spec error(String.t(), Integer.t()) :: Phase.Error.t()
+  defp error(operation_name, operation_count) do
     %Phase.Error{
       phase: __MODULE__,
-      message: error_message()
+      message: error_message(operation_name, operation_count)
     }
   end
 
-  def error_message do
-    ~s(Must provide a valid operation name if query contains multiple operations.)
+  @error_message "Must provide a valid operation name if query contains multiple operations."
```

Comment:
Not sure this needs to be a module attribute, it's static and only used 2 times...

Reply:
I think I copied this from somewhere else in the code but was also unsure about it. Will fix

Reply:
Fixed :)